bugbug/nlp.py

```diff
@@ -30,7 +34,7 @@
         "Spacy model is missing, install it with: "
         f"{sys.executable} -m spacy download en_core_web_sm"
     )
-    print(msg, file=sys.stderr)
+    logger.info(msg)
```

Comment:
Using `error` could be a better fit than `info` in this case. Also, please inline the message.